CHANGELOG.md

```diff
@@ -1,26 +1,268 @@
+It introduces new extension points for the Buildpack, Peregrine, and Venia UI library components.
+
+Developer can use these extension points to extend their storefront project without duplicating and maintaining PWA-Studio code.
+
+As of 7.0.0, PWA-Studio contains the following extension points:
+
+#### Venia UI extension points
+
+Venia UI extension points are declared in [`venia-ui-declare.js`][].
+
+| Target name            | Description                                           |
+| ---------------------- | ----------------------------------------------------- |
+| `richContentRenderers` | Add custom rich content renderers to your storefront. |
+| `routes`               | Add or modify storefront-specific routes              |
+
+#### Peregrine extension points
+
+Peregrine extensions points are declared in [`peregrine-declare.js`][].
+
+| Target name | Description                                                                    |
+| ----------- | ------------------------------------------------------------------------------ |
+| `talons`    | Intercept specific Peregrine talons and wrap them with your custom components. |
+
+Wrappable talons:
+
+- `useProductFullDetail()`
+- `useApp()`
+
+#### Buildpack extension points
+
+Buildpack extension points are declared in [`declare-base.js`][].
+
+| Target name        | Description                                                                    |
+| ------------------ | ------------------------------------------------------------------------------ |
+| `enVarDefinition`  | Add custom environment variables to PWA-Studio's environment variables system. |
+| `transformModules` | Apply custom file transformers through Webpack.                                |
+| `webpackCompiler`  | Access the webpack compiler object.                                            |
+| `specialFeatures`  | Specify special features to the webpack compiler for components.               |
+
+### New Venia look
+
+Release 7.0.0 introduces numerous improvements to the shopper experience with various stylistic changes to the Venia example storefront.
+These storefront changes are the result of extensive research by the UX team to provide an optimal shopping experience.
+
+These improvements are available to developers as individual library components or as a whole when they set up a new storefront project.
+
+#### Page Builder home page
+
+In addition to the style changes for Venia, release 7.0.0 also adds a new home page built using [Page Builder][].
+The content of this CMS page is defined in the Magento Admin using the Page Builder extension.
+It showcases Page Builder content types such as Sliders, Banners, and Gallery Items.
+
+Check it out: https://develop.pwa-venia.com/venia-new-home
+
+### Standalone Cart and Checkout pages
+
+This release builds on the standalone shopping cart page introduced in 6.0.0 and connects it to a new standalone checkout page.
```

Comment:
We should be consistent in capitalizing (or not capitalizing) the adjectives attached to "page" for the various page types. We also need to consider whether or not to capitalize "page". For example, we need to either pick "Standalone Shopping Cart Page", "Standalone Shopping Cart page", or "standalone shopping cart page" (likewise for "standalone checkout page"). There are several examples of this throughout this doc that should be fixed.

Reply:
I went with: 'standalone Shopping Cart page' and 'standalone Checkout page'